static/app/views/profiling/flamegraph.tsx

```diff
@@ -9,4 +9,4 @@ function FlamegraphView() {
   );
 }
 
-export {FlamegraphView};
+export default FlamegraphView;
```

Comment:
Do we need to change this because of how the route is loaded?

Reply:
yes, the route loads the default export